commands/member/cabbages.js

```diff
@@ -1,13 +1,146 @@
-const { SlashCommandBuilder } = require('discord.js');
+const { SlashCommandBuilder, EmbedBuilder } = require('discord.js');
+const { getDiscordClient } = require('../../discord');
+const {
+    getCabbageBreakdown,
+    cabbagesUntilNext,
+} = require('../../helpers/calculateCabbages');
 const models = require('../../models');
 
+const findEmoji = (client, name) => {
+    const emoji = client.emojis.cache.find(
+        (emoji) => emoji.name.toLowerCase() === name.toLowerCase()
+    );
+    return emoji || '';
+};
+
+const capitalize = (string) => {
+    return string.charAt(0).toUpperCase() + string.slice(1).toLowerCase();
+};
+
+const embedfield = (name, value, inline) => ({
+    name: name || ' ',
+    value: value || ' ',
+    inline: inline || false,
+});
+
+const cabbageEmbed = (member, memberData) => {
+    const client = getDiscordClient();
+    const { accountProgression: account } = memberData;
+    // Generate all neccesary info
+    const checkmark = findEmoji(client, 'check');
+    const rankEmojiName = `${memberData.currentRank
+        .toLowerCase()
+        .replace(/ /g, '')}Gem`;
+    const rankEmoji = findEmoji(client, rankEmojiName);
+    const cabbages = Math.floor(memberData.currentCabbages);
+    const cabbageBreakdown = getCabbageBreakdown(memberData);
+    const timestamp = Math.floor(Date.parse(memberData.updatedAt) / 1000);
+    // Generate embed fields
+    const achievementText = [];
+    const statusText = [];
+    const cabbagesText = [];
+    // TODO: Do all these if-statements in a loop, info like achievement text,
+    // emojiname, statustext will have to be saved elsewhere
+    const emoji1 = findEmoji(client, 'cabbageclassic');
```

Comment:
If you are doing `emoji1` up until `emoji8`, I rather have `cabbageEmoji` ... `clogEmoji` etc.

Reply:
Sure 😅

Reply:
dd8cb9e